starters/next-wordpress-starter/components/page.jsx

```diff
@@ -4,6 +4,23 @@ import Link from "next/link";
 export default function Page({
   page: { title, date, featuredImage, content },
 }) {
+  let rxSrc, imgSrc, rxPath, imgPath;
+  let srcUrl = "";
+  try {
+    rxSrc = new RegExp(".+/(.+.+$)"); //We will get the file name
+    imgSrc = rxSrc.exec(featuredImage.node.sourceUrl)[1];
+    rxPath = new RegExp("http[s]?:\\/\\/?[^\\/]+\\/(.*[/]{1}.*/).+.+$");
+    imgPath = rxPath.exec(featuredImage.node.sourceUrl)[1];
+    if (process.env.IMAGE_DOMAIN)
```

Comment:
This will always be false since env vars on `process.env` are not made available to the client side unless it is prefixed with `NEXT_PUBLIC`, which is why I suggested a similar approach that the `next-drupal-starter` is taking with the `lib/constants.js` file